www/pfSense-pkg-squid/files/usr/local/pkg/squid.inc

```diff
@@ -1175,7 +1175,7 @@ function squid_resync_general() {
 					// Should never get here
 					$sslproxy_dhparams = "";
 				}
-				file_put_contents($crt_pk, base64_decode($srv_cert['prv']) . base64_decode($srv_cert['crt']));
+				file_put_contents($crt_pk, str_replace("\r\n", "\n", base64_decode($srv_cert['prv']) . base64_decode($srv_cert['crt'])));
```

Comment:
We have a function for this, `unixnewlines()`.

So you'd use `unixnewlines( base64_decode($srv_cert['prv']) . base64_decode($srv_cert['crt']) )` here instead of  `str_replace()`.

Reply:
Perfect, I'll change it. But I'm getting the same error restarting squid without changing the certificate also, so, please, let this PR on hold until I can make more tests.

Reply:
Done